### PR TITLE
Admin Menu updates.

### PR DESCRIPTION
**Pull request by LiamDormon**
Changed if statements to switch cases for better performence in the future, added a heal player option. Might add more later.

### CODE REVIEW

**Comment by LiamDormon** [comment]
I've implemented a pretty simple death system, Currently no way of letting people revive you but you can optionally respawn or wait.

**Review by Dislaik — CHANGES_REQUESTED** [review]
Remove death detection and i launch it!

**Comment by LiamDormon** [comment]
What do you mean by removing death detection? All I've added is a death detection system since currently when you die it just revives you on the spot.

File: Client/Modules/Core/Plague/Zombie.cs [hunk 1]
```diff
@@ -22,6 +22,37 @@ public Zombie()
...
+        }
+
+        private void Respawn()
+        {
+            var Player = PlayerPedId();
+            ClearPedTasks(Player);
+            ClearPedSecondaryTask(Player);
+            ClearPedTasksImmediately(Player);
+            SetEntityHealth(Player, 200);
+            SetEntityCoords(Player, 427.4668f, -979.3f, 30.71004f, true, true, true, false);
```
Comment (Dislaik) [inline Client/Modules/Core/Plague/Zombie.cs:55]: Remove this

**Review by Dislaik — CHANGES_REQUESTED** [review]

File: Client/Modules/Core/Plague/Zombie.cs (hunk 1 above)
Comment (LiamDormon) [reply Client/Modules/Core/Plague/Zombie.cs:55]: May aswell close this PR then since that is pretty much all I've added.

**Pull request closed without merging**